cmd/init.go

```diff
@@ -69,13 +69,14 @@ var InitCmd = &cobra.Command{
 
 func init() {
 	InitCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Deploy Dapr to a Kubernetes cluster")
-	InitCmd.Flags().BoolVarP(&slimMode, "slim", "s", false, "Initialize dapr in self-hosted mode without placement, redis and zipkin containers.")
-	InitCmd.Flags().StringVarP(&runtimeVersion, "runtime-version", "", "latest", "The version of the Dapr runtime to install. for example: v0.1.0")
+	InitCmd.Flags().BoolVarP(&slimMode, "slim", "s", false, "Exclude Placement service, Redis and Zipkin containers from self-hosted installation")
+	InitCmd.Flags().StringVarP(&runtimeVersion, "runtime-version", "", "latest", "The version of the Dapr runtime to install, for example: 1.0.0")
 	InitCmd.Flags().StringVarP(&initNamespace, "namespace", "n", "dapr-system", "The Kubernetes namespace to install Dapr in")
-	InitCmd.Flags().BoolVarP(&enableMTLS, "enable-mtls", "", true, "Enable mTLS in your cluster. Default: true")
-	InitCmd.Flags().BoolVarP(&enableHA, "enable-ha", "", false, "Deploy Dapr in a highly available mode. Default: false")
+	InitCmd.Flags().BoolVarP(&enableMTLS, "enable-mtls", "", true, "Enable mTLS in your cluster")
```

Comment:
How do I easily see and indication of the default values, if these are not in the description?

Reply:
It is in the description, but it is auto appended. I can copy and paste here for you.

Reply:
```txt
./cli init --help
Install Dapr on supported hosting platforms, currently: Kubernetes and self-hosted

Usage:
  dapr init [flags]

Flags:
      --enable-ha                Enable high availability (HA) mode
      --enable-mtls              Enable mTLS in your cluster (default true)
  -h, --help                     Print this help message
  -k, --kubernetes               Deploy Dapr to a Kubernetes cluster
  -n, --namespace string         The Kubernetes namespace to install Dapr in (default "dapr-system")
      --network string           The Docker network on which to deploy the Dapr runtime
      --redis-host string        The host on which the Redis service resides (default "localhost")
      --runtime-version string   The version of the Dapr runtime to install, for example: 1.0.0 (default "latest")
  -s, --slim                     Exclude placement service, Redis and Zipkin containers from self-hosted installation
```